src/baguetteBox.scss

```diff
@@ -32,8 +32,6 @@
 
         figure {
             display: inline;
-            margin: 0;
-            height: 100%; // Opera 12 image stretching fix
```

Comment:
Basically, it looks like this if you want proof:

![1](https://cloud.githubusercontent.com/assets/5721520/14395372/9cb536ac-fdd1-11e5-8b5f-5925fdc79f28.PNG)
![2](https://cloud.githubusercontent.com/assets/5721520/14395374/9e53b4ca-fdd1-11e5-95c1-b606f74aa333.PNG)

That's on Opera 12.15.